src/main/java/com/questhelper/requirements/AbstractRequirement.java

```diff
@@ -45,6 +48,49 @@ public String getTooltip()
 	@Override
 	public void setTooltip(String tooltip)
 	{
-		this.tooltip = tooltip;
+		appendToTooltip(tooltip);
```

Comment:
Not completely sold on this. It makes sense for `setTooltip` to override what is already in the tooltip string.
Thoughts?